packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -36,14 +36,14 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, ProxyAdminOwnedBase, Re
     address private spacer_253_0_20;
 
     /// @notice Semantic version.
-    /// @custom:semver 2.9.0
-    string public constant version = "2.9.0";
+    /// @custom:semver 2.10.0
+    string public constant version = "2.10.0";
```

Comment:
Similar question here. I know we talked about this, but I can't remember

Reply:
Same as above, change to reinitializer causes this